apps/institution-service/cypress/e2e/institutionsFiltersAndSearch.cy.ts

```diff
@@ -0,0 +1,188 @@
+          institution.aggregatorIntegrations.forEach((aggInt) => {
+            expect(aggInt.supports_aggregation).to.be.true;
+            expect(aggInt.supports_identification).to.be.true;
+            expect(aggInt.supports_verification).to.be.true;
+            expect(aggInt.supports_history).to.be.true;
+          });
+        });
+      },
+    );
+  });
+
+  it("gets a list of institutions that have isActive = true", () => {
+    getInstitutionsRequest({
+      integrationFieldFilter: ["isActive"],
+    }).then(
+      (response: Cypress.Response<{ institutions: InstitutionDetail[] }>) => {
+        response.body.institutions.forEach((institution) => {
+          institution.aggregatorIntegrations.forEach((aggInt) => {
+            expect(aggInt.isActive).to.be.true;
+          });
+        });
+      },
+    );
+  });
+
+  it("gets a list of active institutions with mx agg support and supports_history", () => {
+    getInstitutionsRequest({
+      integrationFieldFilter: ["isActive", "supportsHistory"],
+      aggregatorFilter: ["mx"],
+    }).then(
+      (response: Cypress.Response<{ institutions: InstitutionDetail[] }>) => {
+        response.body.institutions.forEach((institution) => {
+          let hasExpectedAttributes = false;
+          institution.aggregatorIntegrations.forEach((aggInt) => {
+            if (aggInt.aggregator.name === "mx") {
+              hasExpectedAttributes =
+                aggInt.isActive || aggInt.supports_history;
+            }
+          });
+          expect(hasExpectedAttributes).to.be.true;
+        });
+      },
+    );
+  });
+
+  it("gets a list of active institutions with mx agg support and supports_history and has OAuth", () => {
+    getInstitutionsRequest({
+      integrationFieldFilter: ["isActive", "supportsHistory", "supportsOauth"],
+      aggregatorFilter: ["mx"],
+    }).then(
+      (response: Cypress.Response<{ institutions: InstitutionDetail[] }>) => {
+        response.body.institutions.forEach((institution) => {
+          let hasExpectedAttributes = false;
+          institution.aggregatorIntegrations.forEach((aggInt) => {
+            if (aggInt.aggregator.name === "mx") {
+              hasExpectedAttributes =
+                aggInt.isActive ||
+                aggInt.supports_history ||
+                aggInt.supports_oauth;
+            }
+          });
+          expect(hasExpectedAttributes).to.be.true;
+        });
+      },
+    );
+  });
+
+  it("gets a list of active institutions with mx agg support, supports_history, has OAuth and 'Bank' in the name", () => {
+    getInstitutionsRequest({
+      integrationFieldFilter: ["isActive", "supportsHistory", "supportsOauth"],
+      aggregatorFilter: ["mx"],
+      search: "Bank",
+    }).then(
+      (response: Cypress.Response<{ institutions: InstitutionDetail[] }>) => {
+        response.body.institutions.forEach((institution) => {
+          expect(institution.name).to.include("Bank");
```

Comment:
Is the search case sensitive?

Reply:
Yeah because the `iLike` operator is a case insensitive search